pkg/services/ngalert/notifier/alerts.go

```diff
@@ -27,6 +27,10 @@ func (am *Alertmanager) GetAlerts(active, silenced, inhibited bool, filter []str
 		res = apimodels.GettableAlerts{}
 	)
 
+	if !am.initialised {
```

Comment:
Maybe this should be an error? Since this service is not available yet, sounds like a 503 to me.

Reply:
It will send a 503 now